Jenkinsfile.broker

```diff
@@ -47,7 +47,7 @@ pipeline {
                         jobs = ['adamSensors', 'ATAOS', 'ATDome', 'ATDomeTrajectory', 'ATHexapod', 'ATMCSsimulator', 'ATPneumaticssimulator', 'ATSpectrograph', 'ATWhiteLightSource', 
                             'CBP', 'DIMM', 'DSM', 'EAS', 'Electrometer', 'ESS', 'ExternalScripts', 'FiberSpectrograph', 'GenericCamera', 'hexrotcomm', 
                             'M2simulator', 'MTDome', 'MTDomeTrajectory', 'MTEEC', 'MTHexapod', 'MTMount', 'salkafka', 'SimActuators', 'StandardScripts',  
-                            'TunableLaser', 'Watcher', 'WeatherStation', 'ObservatoryControl', 'MTRotator']
+                            'TunableLaser', 'Watcher', 'WeatherStation', 'ObservatoryControl', 'MTRotator', 'PMD']
```

Comment:
Please move PMD to the correct alphabetical position.  ObsContrl and MTRotator need to be last for timing and resource reasons.